[Bug] Test case error.

<!-- Please use this template while reporting a bug and provide as much info as possible. Not doing so may result in your bug not being addressed in a timely manner. Thanks!

-->


**What happened**:

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:


```
=== RUN   TestSimpleTenantManager
    tenant_test.go:48: 
        	Error Trace:	/home/runner/work/arana/arana/pkg/security/tenant_test.go:48
        	Error:      	Not equal: 
        	            	expected: []string{"arana-tenant", "fake-tenant"}
        	            	actual  : []string{"fake-tenant", "arana-tenant"}
        	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1,4 +1,4 @@
        	            	 ([]string) (len=2) {
        	            	- (string) (len=12) "arana-tenant",
        	            	- (string) (len=11) "fake-tenant"
        	            	+ (string) (len=11) "fake-tenant",
        	            	+ (string) (len=12) "arana-tenant"
        	            	 }
        	Test:       	TestSimpleTenantManager
--- FAIL: TestSimpleTenantManager (0.00s)
```

--- a/pkg/security/tenant_test.go
+++ b/pkg/security/tenant_test.go
@@ -45,7 +45,9 @@ func TestSimpleTenantManager(t *testing.T) {
 	tm.PutCluster("fake-tenant", "fake-cluster")
 	tm.PutUser("fake-tenant", &config.User{Username: "fake-user"})
 	tm.PutUser("arana-tenant", &config.User{Username: "arana-user"})
-	assert.Equal(t, tm.GetTenants(), []string{"fake-tenant", "arana-tenant"})
+	tenants = tm.GetTenants()
+	assert.Len(t, tenants, 2)
+	assert.Contains(t, tenants, "fake-tenant", "arana-tenant")
 
 	var (
 		user     *config.User

